src/JBrowse/Store/RemoteBinaryFile.js

```diff
@@ -234,9 +242,14 @@ return declare( null,
         };
 
         req.onreadystatechange = dojo.hitch( this, function() {
-            if (req.readyState == 4) {
+            if (req.readyState == 2) {
+                if(!Util.isElectron() && req.status == 200 && request.expectRanges == true) {
+                    req.abort();
```

Comment:
Have you checked browser support for `req.abort`?

Reply:
It looks like it is pretty widely supported, goes back to ie7 even https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/abort (it appears different from cancellable promise which is a  more weird thing it seems)

Reply:
👍